docs/docs-content/release-notes/known-issues.md

```diff
@@ -16,6 +16,8 @@ The following table lists all known issues that are currently active and affecti
+| Hosts provisioned in [agent mode](../deployment-modes/agent-mode/agent-mode.md) do not display host information in the console after the Palette Terminal User Interface completes host setup.                                                                                                                                                                                                                                                                                                                                                                                            | Local UI is still available and will display host information. Refer to [Access Local UI](../clusters/edge/local-ui/host-management/access-console.md) to learn how to access Local UI.                                                                                                                                                                                                                                                                                                                                                                                                                         | December 12, 2024  | Edge                         |
+| In a multi-node cluster, resetting an in-use follower node does not update the node status on the leader node's linking screen.                                                                                                                                                                                                                                                                                                                                                                                                                                                           | [Scale down](../clusters/edge/local-ui/cluster-management/scale-cluster.md#scale-down-a-cluster) the cluster and free up the follower node before resetting the node.                                                                                                                                                                                                                                                                                                                                                                                                                                           | December 12, 2024  | Edge                         |
```

Comment:
We can change "resetting an in-use follower" -> "the Edge Host reset action on a cluster node". It applies to all cluster nodes.